### PR TITLE
Update SPONSORS.md

### PR DESCRIPTION
**Pull request by gyto6**
Acceptance Criteria:

- [ ] **Features and improvements are fully implemented** so that they can be released at any time without additional work
- [ ] **Automated unit and/or acceptance tests have been added** to ensure the changes work as expected and to reduce repetitive manual work
- [ ] **User interface changes are fully responsive** and have been tested on all major browsers and various devices
- [ ] Database-related changes are compatible with SQLite and MariaDB
- [ ] Translations have been / will be updated (specify if needed)
- [ ] Documentation has been / will be updated (specify if needed)
- [ ] Contributor License Agreement (CLA) has been signed

### CODE REVIEW

**Comment by lastzero** [comment]
Thank you! We were sure to have asked all of our Gold Sponsors if they would like to be listed.... Sorry about that.

**Comment by gyto6** [comment]
No problem, you can"t be everywhere. Your work is more important on the project itself. ;)

File: SPONSORS.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ Garn Penrod (Patreon, November 2020)
 
 [Łukasz Anwajler](https://github.com/anwajler) (GitHub Sponsors, November 2021)
 
+[Gyto6](https://github.com/gyto6) (GitHub Sponsors, Mai 2022)
```
Comment (alexislefebvre) [inline SPONSORS.md:61]: It should be “May 2022”.

Comment (lastzero) [reply SPONSORS.md:61]: Already fixed

**Pull request merged**